### ISSUE TITLE
Port to Pytorch Lightning Format

### ISSUE THREAD

**Issue by karanchahal** [jiasenlu/vilbert_beta#32]
Hello,
We love your repo and thanks for open sourcing it. Are there plans to port this to Pytorch lightning ? We are trying to build upon Vilbert for the CVR task and wanted to know what we will need to change to run a pretrained model on the validation set of VCR.

Thanks !

**Comment by shijing001** [comment]
have been able to replicate this task VCR on your computer?
